test/NuGet.Tests.Apex/NuGet.Tests.Apex.Daily/NuGetConsoleTestCase.cs

```diff
@@ -166,22 +181,30 @@ public async Task UpdatePackageForMauiProjectInPMC(ProjectTemplate projectTempla
 
                     // Assert
                     VisualStudio.AssertNuGetOutputDoesNotHaveErrors();
-                    CommonUtility.AssertPackageInAssetsFile(VisualStudio, testContext.Project, packageName, v200, Logger);
+                    if (projectTemplate.ToString().Equals("WebSiteEmpty"))
+                    {
+                        CommonUtility.AssertPackageInPackagesConfig(VisualStudio, testContext.Project, packageName, v200, Logger);
+                    }
+                    else
+                    {
+                        CommonUtility.AssertPackageInAssetsFile(VisualStudio, testContext.Project, packageName, v200, Logger);
+                    }
                     Assert.IsTrue(VisualStudio.HasNoErrorsInOutputWindows());
                 }
             }
         }
 
         [DataTestMethod]
-        [DynamicData(nameof(GetMauiTemplates), DynamicDataSourceType.Method)]
+        [DataRow(ProjectTemplate.MauiClassLibrary)]
+        [DataRow(ProjectTemplate.WebSiteEmpty)]
         [Timeout(DefaultTimeout)]
-        public async Task UninstallPackageForMauiProjectInPMC(ProjectTemplate projectTemplate)
+        public async Task UninstallPackageInPMC(ProjectTemplate projectTemplate)
```

Comment:
I see in this method (and the other maui tests), there's wait for auto restore etc. 

Would it be better if we just had completely different test cases for websites?

Reply:
> I see in this method (and the other maui tests), there's wait for auto restore etc.
> 
> Would it be better if we just had completely different test cases for websites?

Thanks for your suggestions. I add the test cases for website and adjust the case name.